python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -354,6 +355,19 @@ cdef class Aggregation:
             ))
         return agg
 
+    @classmethod
+    def cov(
+        cls,
+        libcudf_types.size_type min_periods,
+        libcudf_types.size_type ddof=1
+    ):
+        cdef Aggregation agg = cls()
+        agg.c_obj = move(
+            libcudf_aggregation.make_covariance_aggregation[aggregation](
+                min_periods, ddof
+            ))
+        return agg
+
```

Comment:
Right now this isn't actually being used. All the machinery added in this PR other than this change is for groupby aggregation, whereas this particular change is for aggregation on an entire column. If you want to make use of this, you'll have to modify the covariance calculation [here](https://github.com/rapidsai/cudf/blob/branch-22.02/python/cudf/cudf/core/column/numerical_base.py#L174).

Reply:
I apologize, this was an error on my part. I'm not really sure what I was thinking when I wrote this review :joy:. You did the right thing here, we need this defined. Please add it back, then I think your tests should pass.